development/build/index.js

```diff
@@ -41,8 +41,7 @@ function defineAllTasks () {
         manifestTasks.dev,
         reload,
       ),
-    ),
```

Comment:
This change is unexpected. I prefer the older format, and I'm not sure what could have caused this 🤔

Reply:
The first arg to the function is mis-aligned, causing the auto-fix to do this. I'll fix this with db7f5f200.

Reply:
The same problem is present in many other places, but I'm not gonna block on it. Hopefully prettier will fix it.